### DISCUSSION TITLE
why not a "quick courseplay button"

### DISCUSSION THREAD

**Discussion by KonoKinoko** [Courseplay/Courseplay_FS22#1922] [Q&A] [answered]
Hi guys

thank you for your hard work. 

Would be possible to have a "quick courseplay keybindable button". something like, I get to a field, and instead of pressing AI helper, I press quick courseplay, which automatically generate a course for where the tractor is, with the default/favourite settings. 

This would be insanely useful for contract works, where you don't really mind a 100% coverage of a field, but 1-2 headland and standard up/down works without any issue.

**Comment by Tensuko** [comment, marked as answer]
I understand the Idea behind this, but fields and tools are too different.
I dont know how you do it but I drive on field hit on no course, hit generate when I have matching settings from the last time or I adjust stuff, get out if the menu and hit start, just takes a few seconds. Dont see much reasons to make that faster. We are not I a rush... Its still a game.
